documentation/asciidoc/topics/proc_creating_caches_batch.adoc

```diff
@@ -0,0 +1,41 @@
+[id='creating_caches_batches-{context}']
+= Creating Caches in Batches
+Add multiple caches with batch operations with the {brandname} CLI.
```

Comment:
Once the Batch CR PR is up I think we can remove this in favour of it's documentation. Looks good in the interim though (sorry for any duplicated effort here).

Reply:
no, it's OK. I expect this to be replaced by the Batch CR but I would like to get interim docs out there as a means to "prime" users for the Batch CR.